lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +26,63 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class MobileAppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    IOS = "ios"
+    ANDROID = "android"
+    MOBILE_PLATFORM = (
+        (IOS, "IOS"),
+        (ANDROID, "ANDROID"),
+
+    )
+    KEY_FIELDS = ('platform', )  # mobile platform is unique
+    platform = CharField(
+        max_length=50,
+        choices=MOBILE_PLATFORM,
+        blank=False,
+        help_text="mobile device platform")
+    latest_version = TextField(blank=False, help_text="Latest available version for app in X.X.X format")
+    min_supported_version = TextField(blank=False, help_text="Min supported version for app in X.X.X format")
+    next_min_supported_version = TextField(
```

Comment:
_Do you think there is a way of differentiating between these 4 cases if we remove this field?_
Yes.

1. Request is from a version which is not supported (result: return an error).
  `version < min_supported_version AND current_date > update_required_date`
2. Request is from a version which is deprecated and the user needs to upgrade in the near future. (result: tell user they need to upgrade)
  `version < min_supported_version AND current_date <= update_required_date`
3. Request is from a supported but not latest version (result: tell user there is a newer version available).
  `version < latest_version`
4. Request is from the latest version.
  ` version >= latest_version`